doc/background.rst

```diff
@@ -130,7 +130,7 @@ top of lower-level tools, where you can use the top level for anything
 that it provides, while always being able to drop down a level (or
 several if necessary!) to get the behavior you need.
 
-Panel's `interact function <https://panel.pyviz.org/user_guide/Interact.html>`_
+Panel's `interact function <https://panel.holoviz.org/tutorials/intermediate/interactivity.html>`_
```

Comment:
Thanks for the update! Indeed, the old link was out of date, but the suggested one isn't correct either, since that page doesn't describe the `pn.interact` function that's being referred to here. @philippjfr , up to you how you want to handle references to `interact` like this.

Reply:
Ah ok. Is current guidance to use .rx instead of .interact going forward? I thought I saw a document on this.

Happy to update the doc to whatever link / text. But there should still be a 301 to handle visits to the old page from wherever they originate.